src/ldpl.cpp

```diff
@@ -1070,6 +1085,35 @@ bool line_like(string model_line, vector<string> & tokens, compiler_state & stat
         {
             return is_label(tokens[j]);
         }
+        else if(model_tokens[i] == "$math") //$math is a math expression
```

Comment:
This is just awesome, I couldn't have thought of this in a thousand years. I would've just reused $number in a very scruffy manner, this is so much clean. I love your work :laughing: